podium/urls.py

```diff
@@ -20,4 +20,5 @@
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^talks/', include('podium.talks.urls')),
+    url(r'^', include('podium.talks.urls')),
```

Comment:
Rather than including a second URL pattern here, which includes the same set of URL patterns as the line above, considering matching a single empty pattern like `r'^$'` and issuing a redirect to a specific view in an app.  See the url patterns example here: https://docs.djangoproject.com/en/1.11/ref/class-based-views/base/#redirectview
In addition, you can use `reverse_lazy` to ensure the `url` parameter to the `RedirectView` doesn't need to be hardcoded:  https://docs.djangoproject.com/en/1.11/ref/urlresolvers/#reverse-lazy